lib/dialogs/buy/models/BuyDialogEvent.dart

```diff
@@ -32,3 +32,7 @@ class OnConfirmBuy extends BuyDialogEvent {
   @override
   List<Object?> get props => [buyPrice];
 }
+
+class OnErrorEvent extends BuyDialogEvent {
+  List<Object?> get props => [];
```

Comment:
The base `abstract` `BuyDialogEvent` should have the empty list `props` set and then sub-classes like this one doesn't need to override it just to pass an empty list.